unfoldingWord/translationCore
# Fix tests

#### This pull request addresses:

This pull request fixes the tests so that builds can run. 


#### How to test this pull request:

Make sure that the tests on Travis pass.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1049)
